typeshed/__init__.py

```diff
@@ -1,4 +1,53 @@
-import os
+# -*- coding: utf-8 -*-
+"""Typeshed is a collection of type annotations for the standard library and
+third-party libraries.
+
+The type annotations in question are stored in so-called "stub files".  They're
+Python-like files with the *.pyi file extension.  Those files are parseable by
+the built-in Python AST but are not executable.  Functions and methods in stub
```

Comment:
But why would you parse them with the built-in `ast` module? That loses `# type: ignore`. Maybe it's better to just refer to PEP 484 for a discussion of what stubs are? Or the toplevel `README.md`?

Reply:
OK, will change.